packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -73,7 +70,7 @@ const extractAttachmentMetadata = (metadata: Record<string, string>): Attachment
     return [];
   }
 };
-/* @conditional-compile-remove(file-sharing-teams-interop) @conditional-compile-remove(file-sharing-acs) */
+
```

Comment:
do we want to be removing the empty lines as well?

Reply:
I've tried to be consistent with what is already in each respective file throughout